wlauto/resource_getters/standard.py

```diff
@@ -45,6 +45,9 @@ class PackageFileGetter(ResourceGetter):
     Looks for exactly one file with the specified extension in the owner's directory. If a version
     is specified on invocation of get, it will filter the discovered file based on that version.
     Versions are treated as case-insensitive.
+    If an ABI is specified in the resource request, then the getter will try to locate the file in
+    the ABI-specific folder in the form ``<root>/<file_extension>/<abi>/<filename>``. Where ``root``
+    is the base resource location e.g. ``~/.workload_automation/dependencies/<extension_name>``
```

Comment:
`get()` method is three lines, one of which needs to change; so you end up duplicating just two lines of code; to me, this seems preferable to the additional complexity of method 2